### PR TITLE
add information in survival tab titles

### PR DESCRIPTION
**Pull request by dippindots**
add info button for survival tab titles.
Fix # https://github.com/cbioportal/cbioportal/issues/6115.

proposed changes:
a. add the description to the title when there is only one description
image

b. show a warning when the survival data come from multiple studies
image

c. show a warning with additional information when the survival data has different creteria
image

### CODE REVIEW

File: src/pages/resultsView/survival/SurvivalTab.tsx [hunk 1]
```diff
@@ -54,6 +58,18 @@ export default class SurvivalTab extends React.Component<ISurvivalTabProps, {}>
         }
     });
 
+    private makeSurvivalInfoIcon(survivalType: String): JSX.Element {
+        return (
+            <DefaultTooltip
+                placement='top'
+                overlay={`${survivalType} since time of sequencing`}
```
Comment (jjgao) [inline src/pages/resultsView/survival/SurvivalTab.tsx]: We should use the description of the OS_STATUS or DFS_STATUS attribute for the tooltip.

Comment (jjgao) [reply src/pages/resultsView/survival/SurvivalTab.tsx]: image

https://cbioportal-frontend-pr-2345.herokuapp.com/study/summary?id=coadread_tcga_pub

**Comment by jjgao** [comment]
Please also do this for the comparison page.

File: src/pages/resultsView/survival/SurvivalTab.tsx (hunk 1 above)
Comment (dippindots) [reply src/pages/resultsView/survival/SurvivalTab.tsx]: Sure, will use the description from the attribute.

**Comment by dippindots** [comment]
> Please also do this for the comparison page.

Ok! Will add it in this pr.

File: src/pages/resultsView/survival/SurvivalTab.tsx [hunk 2]
```diff
@@ -54,6 +58,18 @@ export default class SurvivalTab extends React.Component<ISurvivalTabProps, {}>
         }
     });
 
+    private makeSurvivalInfoIcon(survivalType: String): JSX.Element {
```
Comment (alisman) [inline src/pages/resultsView/survival/SurvivalTab.tsx]: @computed?

File: src/pages/groupComparison/GroupComparisonStore.ts [hunk 3]
```diff
@@ -823,6 +824,17 @@ export default class GroupComparisonStore {
         }
     }, []);
 
+    readonly activeStudiesClinicalAttributes = remoteData<ClinicalAttribute[]>({
+        await: () => [
+            this.activeStudyIds
+        ],
+        invoke: () => {
+            return client.fetchClinicalAttributesUsingPOST({
+                studyIds:this.activeStudyIds.result!
```
Comment (dippindots) [inline src/pages/groupComparison/GroupComparisonStore.ts]: @alisman Changed to just fetch clinical attribute for all active studies.

File: src/pages/groupComparison/Survival.tsx [hunk 4]
```diff
@@ -115,11 +123,31 @@ export default class Survival extends React.Component<ISurvivalProps, {}> {
...
                     <div style={{marginBottom:40}}>
-                        <h4 className='forceHeaderStyle h4'>{this.overallSurvivalTitleText}</h4>
+                        <div style={{display: 'flex'}}>
+                            <h4 className='forceHeaderStyle h4'>{this.overallSurvivalTitleText}</h4>
+                            {
+                                (overallClinicalAttributes && overallClinicalAttributes.length > 0) && (
+                                    <DefaultTooltip
+                                        placement='top'
+                                        overlay={
+                                            _.reduce(_.uniq(_.map(overallClinicalAttributes, attr => attr.description)), function(allInfo, info) {
```
Comment (alisman) [inline src/pages/groupComparison/Survival.tsx]: use array.join(" / ")

**Comment by alisman** [comment]
@dippindots 
1. we need to move that alert into the div tabMessageContainer to get consistent styling. 
2. I think we need to list different descriptions.  The words "different criteria" could be a tooltip with descriptions listed inside.

**Pull request merged**